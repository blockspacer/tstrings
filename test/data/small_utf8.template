The quick ${color} ${animal} jumps over the lazy ${dog}.
Quick \${escaped} jumps nightly above wizard.
Pack my ${special-char} box with five dozen liquor $jugs.