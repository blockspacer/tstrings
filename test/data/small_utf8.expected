The quiÄ‰k brown fox jumps over the lazy dalmatian.
Quick \${escaped} jumps nightly above wizard.
Pack my ${special-char} box with five dozen liquor $jugs.